pkgcfg/pkgcfg.go

```diff
@@ -19,34 +21,47 @@ type config struct {
 	localConfigPath string            `json:"-"`
 }
 
-func CreateFiles(pkg, basePath string) error {
-	cfg, err := get(pkg)
+func CreateFiles(pkg, rootDir string) error {
+	cfg, err := get(pkg, rootDir)
 	if err != nil {
 		return err
 	}
+	debug.Log("Creating files for package %q create files", pkg)
 	for name, contentPath := range cfg.CreateFiles {
-		filePath := filepath.Join(basePath, name)
+		filePath := filepath.Join(rootDir, name)
+		debug.Log("Creating file %q", filePath)
 		content, err := os.ReadFile(filepath.Join(cfg.localConfigPath, contentPath))
 		if err != nil {
 			return errors.WithStack(err)
 		}
 		if err = createDir(filepath.Dir(filePath)); err != nil {
 			return err
 		}
-		if err := os.WriteFile(filePath, content, 0744); err != nil {
+		t, err := template.New(name + "-template").Parse(string(content))
+		if err != nil {
+			return errors.WithStack(err)
+		}
+		var buf bytes.Buffer
+		if err = t.Execute(&buf, map[string]string{
+			"DevboxRoot":           filepath.Join(rootDir, ".devbox"),
+			"DevboxProfileDefault": filepath.Join(rootDir, ".devbox/nix/profile/default"),
```

Comment:
instead of hardcoding this here, lets reference `devbox.profileDir` (may have to refactor to make it publicly visible)